lua/gitlab/actions/discussions/init.lua

```diff
@@ -795,6 +805,17 @@ M.toggle_draft_mode = function()
   winbar.update_winbar()
 end
 
+---Toggle between draft mode (comments posted as drafts) and live mode (comments are posted immediately)
+M.toggle_sort_by = function()
+  if state.settings.discussion_tree.sort_by == "original_comment" then
+    state.settings.discussion_tree.sort_by = "latest_reply"
+  else
+    state.settings.discussion_tree.sort_by = "original_comment"
+  end
+  u.notify("Sort discussion tree by '" .. state.settings.discussion_tree.sort_by .. "'", vim.log.levels.INFO)
```

Comment:
Rather than having this be a notification, I think this information ought to live in the winbar!

Reply:
I'm thinking just the arrow would work. I'd like to show this state to the user since it's persisted and affects the view of the discussion tree.

It's very possible we'll want to add filtering/searching by text to the winbar, so keeping the sort direction as short as possible is probably best to make room for that.

Reply:
Actually, I'm addressing the winbar width in #435. Maybe have a look at that PR and let me know if you in general agree with the idea. Then we can find a "shrinkable" way for showing the sort order.

Reply:
What about these variants of the description that would be selected based on available space:
1. `↑ by reply` -> `↑ reply` -> `↑`
2.  `↓ by thread` -> `↓ thread` -> `↓`
I think Unicode characters `U+2191` and `U+2193` should not be a problem for any user.

Reply:
"↑ reply" and "↑ thread" look good to me! Clear and short.

Reply:
I've removed the notification and made a couple of small improvements.